### DISCUSSION TITLE
Packaging v0.11.2

### DISCUSSION THREAD

**Discussion by srwareham** [hassio-addons/app-tailscale#213] [General]
@frenck  thank you for the fantastic contributions to the community and this project. Understandable if it's not a priority, but might it be possible to publish the current main branch into a new minor release? Main motivator would be to allow Headscale users to enjoy the addon, which is currently gated behind the merged pull request that immediately followed the last release: https://github.com/hassio-addons/addon-tailscale/issues/193

Thank you again!

**Comment by frenck** [comment]
> but might it be possible to publish the current main branch into a new minor release?

Sure, but if I do that in the current state it will break setups for some people (make their instances fully unreachable). So do you still think that your request is a good idea?

../Frenck

**Comment by srwareham** [comment]
My incorrect assumption was that the main branch held only tested or presumed working commits. Apologies for wasting your time.
